src/restc.erl

```diff
@@ -151,6 +149,24 @@ construct_url(SchemeNetloc, Path, Query) when is_list(SchemeNetloc),
     urlunsplit(S, N, P, Query).
 
 %%% INTERNAL ===================================================================
+accept(Headers, Type) ->
+    case lists:keyfind(<<"Accept">>, 1, Headers) of
```

Comment:
This could result in weirdness if someone sends in for example <<"accept">> (wrong case) as a header, is this handled somewhere? Otherwise we probably need to add a normalization of case for the header keys.
This might or might not be a problem, I'm not 100% sure what the HTTP spec says about case insensitivity of these header keys.

Reply:
Good catch, headers are case insensitive so added a normalize for headers. :)